Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -332,6 +332,14 @@ def SwapKeepAndDrop(l):
 RAWSIMEventContent.outputCommands.extend(IOMCRAW.outputCommands)
 RAWSIMEventContent.outputCommands.extend(CommonEventContent.outputCommands)
 #
+# temporary collections needed for Phase-2 RECO using RAWSIM as input
+#
```

Comment:
wondering if you could put one line of explanation why it's needed, or at least point to issue https://github.com/cms-sw/cmssw/issues/33172

Reply:
@mmusich 
Not sure I understand the comment. I update the comment and PR description. The goal of this PR is to use eventcontents as we use for the production in prod-like workflow. This will keep us a track on what is needed content of RAWSIM/PREMIXRAW.

Reply:
As we discussed in Issue #33172 this is certainly not needed forever. Thanks for updating comment, I would have preferred a more verbose one but fine